spec/validators/qc_receptions_factory_validator_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe QcReceptionsFactoryValidator do
+  describe '#validate' do
+    let(:used_by) { 'tol' }
+    let(:record) { build(:qc_receptions_factory) }
+
+    before do
+      create(:qc_assay_type, key: 'sheared_femto_fragment_size', label: 'Sheared Femto Fragment Size (bp)', used_by: 2, units: 'bp')
```

Comment:
Maybe create an QcAssayType for `used_by: 0` too, to make sure only the relevant assay types are being used

Reply:
Done